python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1319,8 +1420,53 @@ def from_tensorflow(self, graph, layout="NHWC", shape=None, outputs=None):
                         inputs.append(self._nodes[i][0])
                         input_shapes[self._nodes[i][0]] = self._output_shapes[i]
                 attr['_input_shapes'] = input_shapes
+                node_name_prefix = node.name.rsplit('/', 1)[0]
 
-                op = self._convert_operator(node.op, inputs, attr, graph)
+                if node.op == "Merge":
```

Comment:
Can we move the control ops handling outside to keep the main function simple ?
It would be great if we can make them similar to other ops or move the implementation to ```_convert_control_flow_operator``` with in ```GraphProto``` scope.

Reply:
Done